src/main/java/org/jitsi/videobridge/octo/OctoTentacle.java

```diff
@@ -302,7 +388,8 @@ public void expire()
         logger.info("Expiring");
         setRelays(new LinkedList<>());
```

Comment:
If you make this `Collections.emptyList()` you'll get an immutable list, which ensures no relays are added to an expired tentacle.

Reply:
We always reassign a new value though (as opposed to adding and removing)--though it should use `Collections.emptyList` anyway.